alembic/versions/36a79ffcfbfb_add_first_name_last_name_fullname_.py

```diff
@@ -0,0 +1,27 @@
+"""add first_name, last_name, fullname, initials
+
+Revision ID: 36a79ffcfbfb
+Revises: fecf1191b6f0
+Create Date: 2019-06-28 15:21:50.893256
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '36a79ffcfbfb'
+down_revision = 'fecf1191b6f0'
```

Comment:
`down_revision = 'bafdcae12f97'`

Reply:
good catch, by you and by our trusty test suite friend